data/relation-gardony-kozpont.yaml

```diff
@@ -5,6 +5,12 @@ filters:
       - {start: '1', end: '99'}
   Alkotmány utca:
     ranges: []
+  Bikavölgyi út:
+    # 168/42: ez helyrajzi szám, nem házszám.
+    invalid: ['168/42']
```

Comment:
A validator a / után nem enged több számot, de sztem ezt lazítani kéne, jogos az igény, hogy hrszeket lehessen szűrni egyesével is.